core/src/test/scala/org/broadinstitute/dsde/rawls/user/UserServiceSpec.scala

```diff
@@ -291,207 +284,19 @@ class UserServiceSpec
-      verify(mockSamDAO).deleteResource(SamResourceTypeNames.billingProject, project.projectName.value, testContext)
-      verify(mockSamDAO).deleteResource(SamResourceTypeNames.googleProject, project.googleProjectId.value, testContext)
-      verify(mockGcsDAO, never()).deleteV1Project(project.googleProjectId)
-
-      runAndWait(rawlsBillingProjectQuery.load(defaultBillingProjectName)) shouldBe empty
-      actual shouldEqual ()
-    }
-  }
-
-  it should "fail with a 400 when workspace exists in this billing project to be deleted" in {
-    withEmptyTestDatabase { dataSource: SlickDataSource =>
-      val project = defaultBillingProject
-      // A workspace with the which namespaceName equals to defaultBillingProject's billing project name.
-      val workspace = Workspace(
-        defaultBillingProjectName.value,
-        testData.wsName.name,
-        UUID.randomUUID().toString,
-        "aBucket",
-        Some("workflow-collection"),
-        currentTime(),
-        currentTime(),
-        "test",
-        Map.empty
-      )
-
-      runAndWait(rawlsBillingProjectQuery.create(project))
-      runAndWait(workspaceQuery.createOrUpdate(workspace))
-
-      val mockSamDAO = mock[SamDAO](RETURNS_SMART_NULLS)
-      when(
-        mockSamDAO.userHasAction(SamResourceTypeNames.billingProject,
-                                 project.projectName.value,
-                                 SamBillingProjectActions.deleteBillingProject,
-                                 testContext
-        )
-      ).thenReturn(Future.successful(true))
-
-      val userService = getUserService(dataSource, mockSamDAO)
-
-      val actual = intercept[RawlsExceptionWithErrorReport] {
-        Await.result(userService.deleteBillingProject(defaultBillingProjectName), Duration.Inf)
-      }
-      actual.errorReport.statusCode shouldEqual Option(StatusCodes.BadRequest)
-    }
-  }
-
-  it should "fail with a 403 when Sam says the user does not have permission to delete billing project" in {
-    withEmptyTestDatabase { dataSource: SlickDataSource =>
-      val project = defaultBillingProject
-      runAndWait(rawlsBillingProjectQuery.create(project))
-
-      val mockSamDAO = mock[SamDAO](RETURNS_SMART_NULLS)
-      when(
-        mockSamDAO.userHasAction(SamResourceTypeNames.billingProject,
-                                 defaultBillingProjectName.value,
-                                 SamBillingProjectActions.deleteBillingProject,
-                                 testContext
-        )
-      ).thenReturn(Future.successful(false))
-
-      val userService = getUserService(dataSource, mockSamDAO)
-
-      val actual = intercept[RawlsExceptionWithErrorReport] {
-        Await.result(userService.deleteBillingProject(defaultBillingProjectName), Duration.Inf)
-      }
-      actual.errorReport.statusCode shouldEqual Option(StatusCodes.Forbidden)
-    }
-  }
 
   it should "not delete the Google project when unregistering a Billing project" in {
```

Comment:
nit: test should be renamed since there is no longer a Google project to delete when unregistering. Maybe "delete the billing project records from Sam and Rawls when unregistering a billing project"